src/appengine/handlers/cron/project_setup.py

```diff
@@ -161,10 +165,16 @@ def job_name(self, project_name, config_suffix):
             'address': LIBFUZZER_ASAN_I386_JOB,
             'none': LIBFUZZER_NONE_I386_JOB,
         },
+        'arm': {
+            'hardware': LIBFUZZER_HWASAN_JOB,
+        }
     },
     'afl': {
         'x86_64': {
             'address': AFL_ASAN_JOB,
+        },
```

Comment:
nit: In Oss-fuzz we only try to make sure each target is fuzzed with AFL in one configuration (e.g. libfuzzer has x86 and x64 builds but afl only has x86). I wouldn't add more AFL configs otherwise, it can become a headache for devs/maintainers to deal with and has limited benefit.

Reply:
Done.